rtlib/table.hh

```diff
@@ -447,7 +451,8 @@ namespace Table {
       if (a>b)
         std::swap(a, b);
 
-      //std::cerr << "i " << i << " j " << j << " a " << a << " b " << b << std::endl;
+      // std::cerr << "i " << i << " j " << j << " a " <<
+      // a << " b " << b << std::endl;
```

Comment:
I'm not sure if I like this style, personally. To split a single-line comment into two lines because - I assume - the comment is considered too long? But that's just personal preference.

Reply:
I personally agree, but for (historic) reasons, source code lines should never exceed 80 characters. I guess this is due to the fact that early day's terminals were exactly 80 characters wide and if you think of tools like `diff` it makes sense to have finite numbers of chars in a line. I would prefer a larger cut-off, but if the world picks 80 than let's go with it.